application/templates/application/application-overview.html

```diff
@@ -0,0 +1,140 @@
+                        {% if form.date_of_birth.help_text %}
+                        <p class="help">{{ form.date_of_birth.help_text|safe }}</p>
+                        {% endif %}
+                    </div>
+                    <div class="col-auto my-1 col-md-6">
+                        {{ form.gender }}
+                        <label class="mr-sm-2 sr-only" for={{ form.gender.id_for_label }}>{{ form.gender.label }}</label>
+                        {% if form.gender.help_text %}
+                        <p class="help">{{ form.gender.help_text|safe }}</p>
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.address }}
+                        <label for={{ form.address.id_for_label }}>{{ form.address.label }}</label>
+                        {% if form.address.help_text %}
+                        <p class="help">{{ form.address.help_text|safe }}</p>
+                        {% endif %}
+                    </div>
+                    <div class="form-label-group col-md-6">
+                        {{ form.gpa }}
+                        <label for={{ form.gpa.id_for_label }}>{{ form.gpa.label }}</label>
+                        {% if form.gpa.errors %}
+                        {{ form.gpa.errors }}
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.parent_name }}
+                        <label for={{ form.parent_name.id_for_label }}>{{ form.parent_name.label }}</label>
+                        {% if form.parent_name.errors %}
+                        {{ form.parent_name.errors }}
+                        {% endif %}
+                    </div>
+                    <div class="form-label-group col-md-6">
+                        {{ form.parent_phoneNumber }}
+                        <label for={{ form.parent_phoneNumber.id_for_label }}>{{ form.parent_phoneNumber.label }}</label>
+                        {% if form.parent_phoneNumber.errors %}
+                        {{ form.parent_phoneNumber.errors }}
+                        {% endif %}
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.school }}
+                        <label for={{ form.school.id_for_label }}>{{ form.school.label }}</label>
+                        {% if form.school.errors %}
+                        {% endif %}
+                    </div>
+                    <div class="form-label-group col-md-6">
+                        {{ form.program }}
+                        <label for={{ form.username.id_for_label }}>{{ form.program.label }}</label>
```

Comment:
The use of "form.username" instead of "form.program" should be either rectified or commented about the reason for the usage (same as application-form.html).